keps/sig-auth/2799-reduction-of-secret-based-service-account-token/README.md

```diff
@@ -122,6 +122,11 @@ tokens which are [less secure than the bound token](https://github.com/kubernete
   2. Emit warnings when using auto-generated token secrets.
   3. Add pointers of TokenRequest API and manually created tokens in the validation
      result.
+  4. Marked the auto-generated tokens as invalid if they are not used for more
+     than one year.
```

Comment:
reference the `--legacy-service-account-token-clean-up-period` option and indicate it *defaults* to one year

Reply:
Done.